consensus/src/runway/packer.rs

```diff
@@ -113,16 +113,16 @@ mod tests {
     const NODE_ID: NodeIndex = NodeIndex(0);
     const N_MEMBERS: NodeCount = NodeCount(4);
 
-    fn prepare(
-        keychain: Keychain,
-    ) -> (
+    type Preliminaries = (
```

Comment:
Cute!

Reply:
Although it would probably be nicer as a `struct`, then you could use `..` when unpacking it and the names would get standardized between tests, improving quality.